umm-spec-lib/src/cmr/umm_spec/umm_g/granule.clj

```diff
@@ -57,19 +58,19 @@
      {:granule-ur (:GranuleUR umm-g-json)
       :data-provider-timestamps (umm-g->DataProviderTimestamps umm-g-json)
       :collection-ref coll-ref
-      ; :data-granule (xml-elem->DataGranule umm-g-json)
-      ; :access-value (cx/double-at-path umm-g-json [:RestrictionFlag])
       :temporal (umm-g->Temporal umm-g-json)
-      ; :orbit-calculated-spatial-domains (ocsd/xml-elem->orbit-calculated-spatial-domains umm-g-json)
       :platform-refs (platform/umm-g-platforms->PlatformRefs (:Platforms umm-g-json))
       :project-refs (project/umm-g-projects->ProjectRefs (:Projects umm-g-json))
       :cloud-cover (:CloudCover umm-g-json)
-      ; :two-d-coordinate-system (two-d/xml-elem->TwoDCoordinateSystem umm-g-json)
-      ; :related-urls (ru/xml-elem->related-urls umm-g-json)
-      ; :spatial-coverage (xml-elem->SpatialCoverage umm-g-json)
+      :two-d-coordinate-system (tiling-system/umm-g-tiling-identification-system->TwoDCoordinateSystem
+                                (:TilingIdentificationSystem umm-g-json))})))
```

Comment:
Please do not move around the lines or parentheses. Just uncomment out the field that you are changing to minimize the merge conflict with other people also making changes to UMM-G.

Reply:
Let me change it back to the order it was then.